specifyweb/frontend/js_src/lib/components/FormSliders/RecordSelectorFromCollection.tsx

```diff
@@ -103,7 +105,7 @@ export function RecordSelectorFromCollection<SCHEMA extends AnySchema>({
       const resources = isToOne ? rawResources.slice(0, 1) : rawResources;
       if (isDependent && isToOne)
         collection.related?.placeInSameHierarchy(resources[0]);
-      if (!isInteraction) collection.add(resources);
+      if (!isInteraction && formType !== 'formTable') collection.add(resources);
       handleAdd?.(resources);
```

Comment:
could you please refactor the code such that this component does not receive the isInteraction and formType props at all? (generally, try to reduce how many props a component receives, as that both improves maintainability of the code and improves performance due to fewer re-renders)
it should be pretty simple - just move this line that you touched:

```ts
if (!isInteraction && formType !== 'formTable') collection.add(resources);
```

into the handleAdd